app/utils/queue.js

```diff
@@ -14,7 +14,7 @@ export default class Queue {
 
       if (differenceInDays >= 30) {
```

Comment:
@hengsokly 30 should be store in the environment variable where it's aware more than